ADO/ADOLibrary/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,184 @@
+﻿using ADOLibrary.Interfaces;
+using ADOLibrary.Models;
+using System.Data.SqlClient;
+using System.Data;
+using Z.BulkOperations;
+
+namespace ADOLibrary.Repositories
+{
+    public class OrderRepository : IOrderRepository
+    {
+        private string ConnectionString { get; set; }
+        public OrderRepository(DbConfig dbConfig)
+        {
+            ConnectionString = dbConfig.ConnectionString;
+        }
+        public async Task Delete(int id)
+        {
+            using SqlConnection connection = new(ConnectionString);
+            connection.Open();
+
+            var queryString = "DELETE FROM [dbo].[Order] WHERE Id = @Id";
+            var command = new SqlCommand(queryString, connection);
+            command.Parameters.AddWithValue("@Id", id);
+
+            try
+            {
+                await command.ExecuteNonQueryAsync();
+                connection.Close();
```

Comment:
No need it close method as using will take care of it eventually and I'm not sure if try/catch is necessary here (I guess you have it in upper level).